packages/tools/generator-widget/CHANGELOG.md

```diff
@@ -1,12 +1,15 @@
 # Changelog
+
 All notable changes to this widget will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## 9.0.1 - 2021-04-26
+
 ### Fixed
-- Fixes failing `npm install` executions when scaffolding a widget project with the widget generator with NPM v7.x.x.
-- Explains users in the README of generated widget project how to install dependencies with NPM v7.x.x.
-- Set required Node.js version to v12 or higher for the generator itself and the generated widget projects, since Node.js v10 is EOL at the end of April 2021 and pluggable widgets tools requires Node.js v12.
 
-## [major.minor.patch] - YYYY-MM-DD
+-   Fixes failing `npm install` executions when scaffolding a widget project with the widget generator with NPM v7.x.x.
+-   Explains users in the README of generated widget project how to install dependencies with NPM v7.x.x.
+-   Set required Node.js version to v12 or higher for the generator itself and the generated widget projects, since Node.js v10 is EOL at the end of April 2021 and pluggable widgets tools requires Node.js v12.
```

Comment:
It's fine since this PR is small, but these random whitespace changes are really annoying during review 😭 For future PRs, please only include them if they have added value.

Reply:
I will make sure to do these changes in a separate commit or separate PR next time depending on the situation.